### PR TITLE
Allow editing URL in admin

### PR DESCRIPTION
**Pull request by gabebw**
#### Check out that "Edit" link

index

#### Ooooh, a form!

editing

### CODE REVIEW

File: app/views/admin/redirections/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<h1>Editing <code><%= @redirection.slug %></code></h1>
+
+<p>Original URL: <%= @redirection.original_url %></p>
+
+<%= simple_form_for [:admin, @redirection] do |form| %>
+  <%= form.input :url, label: "URL" %>
+  <%= form.button :submit %>
```
Comment (edwardloveall) [inline app/views/admin/redirections/edit.html.erb:7]: Hot Tip! You can do `form.submit` as well.

File: lib/templates/erb/scaffold/_form.html.erb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<%# frozen_string_literal: true %>
+<%%= simple_form_for(@<%= singular_table_name %>) do |f| %>
```
Comment (edwardloveall) [inline lib/templates/erb/scaffold/_form.html.erb:2]: TIL about `<%%` and `<%%=` https://stackoverflow.com/a/40487347/638966

File: spec/features/admin_edits_redirection_spec.rb [hunk 3]
```diff
@@ -0,0 +1,24 @@
+require "rails_helper"
+
+RSpec.feature "Admin edits redirection" do
+  scenario "edits URL" do
+    new_url = "https://example.com/new"
+    url = "https://foobar.neocities.org"
+    original_url = "https://original.neocities.org"
+    slug = "binturong"
+    redirection = create(:redirection, slug:, url:, original_url:)
```
Comment (edwardloveall) [inline spec/features/admin_edits_redirection_spec.rb:9]: Is this a shortcut for `create(:redirection, slug: slug, etc)`? I've never seen blank keyword arguments when calling a method.

**Review by edwardloveall — APPROVED** [review]

File: lib/templates/erb/scaffold/_form.html.erb (hunk 2 above)
Comment (gabebw) [reply lib/templates/erb/scaffold/_form.html.erb:2]: Actually, let's just remove this. We don't really use scaffolding.

File: spec/features/admin_edits_redirection_spec.rb (hunk 3 above)
Comment (gabebw) [reply spec/features/admin_edits_redirection_spec.rb:9]: So your first comment was correct!

As for your second comment: Maybe you didn't set the local variable name when testing in IRB, or maybe you ran it in an IRB that was older than v3.1.0?

**Pull request merged**